recipes/podofo/all/conanfile.py

```diff
@@ -79,17 +90,27 @@ def requirements(self):
         if self.options.with_unistring:
             self.requires("libunistring/0.9.10")
 
+    def build_requirements(self):
+        if Version(self.version) >= "0.10.4":  # pylint: disable=conan-condition-evals-to-constant
+            self.tool_requires("cmake/[>=3.15.7]")
```

Comment:
CMake needs the upper bound for the version range
```suggestion
            self.tool_requires(""cmake/[>=3.15.7 <4]"")
```